cmd/evm/internal/t8ntool/execution.go

```diff
@@ -388,24 +388,25 @@ func MakePreState(db ethdb.Database, chainConfig *params.ChainConfig, pre *Prest
 			statedb.SetState(addr, k, v)
 		}
 	}
-	// Commit db an create a snapshot from it.
-	mptRoot, err := statedb.Commit(0, false)
-	if err != nil {
-		panic(err)
-	}
-	rawdb.WritePreimages(mptSdb.DiskDB(), statedb.Preimages())
-	mptSdb.TrieDB().WritePreimages()
-	snaps, err := snapshot.New(snapshot.Config{AsyncBuild: false, CacheSize: 10}, mptSdb.DiskDB(), mptSdb.TrieDB(), mptRoot)
-	if err != nil {
-		panic(err)
-	}
-	if snaps == nil {
-		panic("snapshot is nil")
-	}
-	snaps.Cap(mptRoot, 0)
```

Comment:
I'm still debugging, but this change makes no sense to me as it should be used both during the conversion and before the conversion starts (if you run several blocks in non-verkle mode).

I understand that you might want to deactivate it if the conversion has ended, but then I would rather add a check for the end of the conversion here.

I might be missing something, I'm going to keep digging.